source/draw/Color.ooc

```diff
@@ -163,7 +164,7 @@ ColorRgba: cover {
 	mix: static func (a, b: This, ratio: Float) -> This {
 		This new(Float mix(a r, b r, ratio) as Byte, Float mix(a g, b g, ratio) as Byte, Float mix(a b, b b, ratio) as Byte, Float mix(a a, b a, ratio) as Byte)
 	}
-	distance: func (other: This) -> Float { (this toRgb() distance(other toRgb()) * 3.0f + (this a - other a) as Float pow(2)) / 4.0f sqrt() }
+	distance: func (other: This) -> Float { ((this toRgb() distance(other toRgb()) as Float pow(2) * 3.0f + (this a - other a) as Float pow(2)) / 4.0f) sqrt() }
```

Comment:
If you write a logo using alpha instead of monochrome then alpha would be like intensity but it all depends on the use case of the channels.